release_notes/45-release-notes.adoc

```diff
@@ -0,0 +1,421 @@
+
+[id="bring-your-own-pki_{context}"]
+=== Bring your own PKI for signature verification
+
+Before this release, you could only verify image signatures against pre-configured keys. With {product-title-short} 4.5, Red{nbsp}Hat has extended the image signature verification to include verifying certificates. This method is available in addition to verifying public keys, allowing you to bring your own public key infrastructure (PKI) to verify signatures.
+
+[id="build-time-network-policy-tools-updates_{context}"]
+=== Build-time network policy tools updates
+
+This release introduces an update to the `roxctl netpol generate` command.
+
+* By default, the command uses port `53` for DNS connections.
+* You can use the `--dnsport` option to override the default DNS port. If you are using {ocp}, you must always change the port when generating network policies with `roxctl` because {ocp} uses port `5353` by default. For example, `roxctl netpol generate --dnsport 5353 <other-options>`.
+
+To simplify usage for {ocp} and other systems that use named ports, Red{nbsp}Hat plans to enhance the `--dnsport` option in a future release to accept strings and numbers. With that change, the generated network policy can use a named port, such as `dns`, instead of a specific port number, which is a more portable approach.
+
+For more details, see xref:../cli/generating-build-time-network-policies.adoc#using-the-build-time-network-policy-generator_generating-build-time-network-policies[Generating build-time network policies].
+
+[id="enhanced-acscs-experience_{context}"]
+=== Enhanced {product-title-managed-short} experience
+
+We have improved the {product-title-managed-short} experience on `console.redhat.com`, making signing up for a trial and getting support easier.
+
+[id="notable-technical-changes_{context}"]
+== Notable technical changes
+
+* For {product-title-short} Operator, Red{nbsp}Hat has added a label selector for caching configuration of secrets and configmaps, resulting in a significant reduction in memory consumption, especially on large clusters. In testing, we have seen a 28% decrease in memory usage on a new {ocp} cluster.
+
+* The {product-title-short} Operator now adds a label `app.stackrox.io/managed-by: operator` to all Helm chart resources and secrets created by the operator, providing better organization and visibility.
+
+* The {product-title-short} Operator increases the number of requests sent to the API server to retrieve secrets not managed by the Operator and do not match the cache label selector.
+
+* Scanner DB now runs on PostgreSQL 15, replacing the earlier version, PostgreSQL 12. Since the database is not persisted, no migration is required, and you can continue using the scanner without any additional steps.
```

Comment:
```suggestion
* Scanner DB now runs on PostgreSQL 15, replacing the earlier version, PostgreSQL 12. Because the database is not persisted, no migration is required, and you can continue using the scanner without any additional steps.
```
Per IBM Style: "To facilitate translation, use _since_ only to refer to time. Do not use it as a synonym for _because_."